__init__.py

```diff
@@ -33,8 +33,8 @@ def __matchVersion():
     if cura_version == "master":
         Logger.log("d", "Running Cura from source. Skipping version check")
         return True
-    if cura_version == "Arachne_engine_alpha":
-        Logger.log("d", "Running Cura Arachne_engine_alpha. Skipping version check")
+    if cura_version == "Arachne_engine_beta":
```

Comment:
I think we need both version checks, because some of the users may use alfa, and some of them use the beta version.
I'd suggest keeping the previous check and add the new one.

Reply:
Yes, this is reasonable, of course, let there be both versions.